### ISSUE TITLE
Duplicate aria-label

### ISSUE THREAD

**Issue by zubairslamdien** [cgkineo/adapt-pageNav#60]
### Aria label needs removing from template
The aria-label on the button container should be removed because it already gets one from the core ariaRegion div.

### Screenshots (if you can)
![image](https://github.com/cgkineo/adapt-pageNav/assets/110032382/a455e849-169a-4335-bc78-4b2aa9848dfe)

**Comment by oliverfoster** [comment]
You could do it the other way round? 

`<nav></nav>` is equivalent to `<div role="navigation"></div>`

[Role navigation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/navigation_role) is a landmark role, so you'd be able to jump to it with accessibility shortcuts. 

It makes sense that the `nav` tag has a label.

Top top aria label can come from either [here for handlebars](https://github.com/adaptlearning/adapt-contrib-core/blob/b67304eff947d2f88d864297a3877aebdb7f1b41/templates/partials/component.hbs#L14) or [here for jsx](https://github.com/adaptlearning/adapt-contrib-core/blob/b67304eff947d2f88d864297a3877aebdb7f1b41/templates/header.jsx#L67-L70). You can see that those functions are controlled with the model attribute `_isA11yComponentDescriptionEnabled` which [defaults to true in adaptModel](https://github.com/adaptlearning/adapt-contrib-core/blob/b67304eff947d2f88d864297a3877aebdb7f1b41/js/models/adaptModel.js#L37).

Probably best to force the pageNav model to default to `false` for `_isA11yComponentDescriptionEnabled` instead of removing from the `nav` element?
